build.sbt

```diff
@@ -34,7 +34,7 @@ libraryDependencies ++= Seq(
   "org.mockito" % "mockito-all" % "1.9.5" % "test",
   
   // All the below are required to force aws libraries to use the latest version of jackson
-  "com.fasterxml.jackson.core" % "jackson-databind" % jacksonVersion,
+  "com.fasterxml.jackson.core" % "jackson-databind" % "2.10.5.1",
```

Comment:
Should all the Jackson libraries be on the same version?

Reply:
Ideally, I tried updating them all and ran into some issues so tried only the one